src/app/app.component.html

```diff
@@ -179,7 +179,7 @@
           <input type="password" name="password" class="form-control" style="width: 100%;"/>
           <br>
           <input type="checkbox" name="showpass" value="password" id="wallet-password"/>
-          <label for="wallet-password">SHOW PASSWORD</label>
+          <label for="wallet-password" style="top: 20;">SHOW PASSWORD</label>
```

Comment:
Can we move the style into a Css class, it'll probably need to change on all the modals